.scala-steward.conf

```diff
@@ -0,0 +1,5 @@
+pullRequests.frequency = "0 10 * * 1-5" #10 am Monday to Friday
```

Comment:
Just so you know, it looks like Scala Steward doesn't like this cron expression! You can see this by searching for `InvalidCron` in the [_Public Repos Scala Steward_](https://github.com/guardian/scala-steward-public-repos/actions/workflows/public-repos-scala-steward.yml) GitHub Action job logs:

![image](https://user-images.githubusercontent.com/52038/214291148-8e288dd6-89de-4dbf-9828-d55aa05cb195.png)

The error in the logs is _"Either DayOfMonth and DayOfWeek must have a ? expression"_:

```
2023-01-24 12:20:47,380 INFO  Refresh cache of guardian/support-frontend
2023-01-24 12:20:47,464 INFO  Failed to parse .scala-steward.conf: cron4s.InvalidCron: Either DayOfMonth and DayOfWeek must have a ? expression: DownField(frequency),DownField(pullRequests)
```

`DayOfMonth` is the 4th value, and `DayOfWeek` is the 5th, instead of:

```
0 10 * * 1-5
```

...I think you probably want:

```
0 10 * ? 1-5
```

An asterisk `*` in the `DayOfMonth` field means "run _every_ day of the month", but that's incompatible with only running on weekdays (the `1-5`), so the question mark `?` is the approved way of denoting that you'll just accept whatever days of the month work with the day-of-week rule you've set, I think.

Reply:
@LAKSHMIRPILLAI points out in https://github.com/guardian/contributions-platform/pull/263 that `0 10 * ? 1-5` fails too!

I do think trying a few things out with the cron4s library in a little project might be the best way to work out what a valid cron is. It's unfortunate that cron syntax isn't completely standardised- there's even inconstancies about how many fields should be used. I can see that we've been using 5 fields, eg `0 10 * * 1-5`, but the examples in the cron4s docs all seem to use 6 fields: https://www.alonsodomin.me/cron4s/userguide/index.html#parsing - 

- Seconds
- Minutes
- Hour Of Day
- Day Of Month
- Month
- Day Of Week